apps/mobile/src/libs/packages/notification/libs/constants/constant.ts

```diff
@@ -0,0 +1,4 @@
+export {
```

Comment:
Either create a separate file for each or define both in this file

Reply:
Agreed, let's make a `DEFAULT_TOAST_MESSAGE = { title: '', message: ''}`